src/components/checkbox/checkbox.ts

```diff
@@ -126,7 +129,9 @@ export class MdCheckbox implements ControlValueAccessor {
 
   hasFocus: boolean = false;
 
-  constructor(private _renderer: Renderer, private _elementRef: ElementRef) {}
+  constructor(private _renderer: Renderer, private _elementRef: ElementRef) {
+    this.change = this._change.asObservable();
```

Comment:
I think we're actually going to go with getters for these. The end result should look something like
```ts
private _change = new EventEmitter<X>();
@Output() get change: Observable<X> {
  return this._change.asObservable();
}
```